udfs/flex/collections/union.md

```
@@ -0,0 +1,69 @@
+# coll.union
+
+## Description
+Combines two lists and returns a new list containing all unique elements from both lists. Duplicates are automatically removed, treating the inputs as sets.
+
+## Syntax
+```cypher
+flex.coll.union(list1, list2)
+```
+
+## Parameters
+| Parameter | Type | Required | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)